src/main/resources/db/migration/V0001__init.sql

```diff
@@ -0,0 +1,482 @@
+CREATE OR REPLACE VIEW sdt_user.bulk_cust_app_seq AS SELECT * FROM sdt_owner.bulk_cust_app_seq;
+CREATE OR REPLACE VIEW sdt_user.bulk_cust_seq AS SELECT * FROM sdt_owner.bulk_cust_seq;
+CREATE OR REPLACE VIEW sdt_user.bulk_submissions AS SELECT * FROM sdt_owner.bulk_submissions;
+CREATE OR REPLACE VIEW sdt_user.bulk_sub_seq AS SELECT * FROM sdt_owner.bulk_sub_seq;
+CREATE OR REPLACE VIEW sdt_user.error_logs AS SELECT * FROM sdt_owner.error_logs;
+CREATE OR REPLACE VIEW sdt_user.error_messages AS SELECT * FROM sdt_owner.error_messages;
+CREATE OR REPLACE VIEW sdt_user.err_log_seq AS SELECT * FROM sdt_owner.err_log_seq;
+CREATE OR REPLACE VIEW sdt_user.err_mesg_seq AS SELECT * FROM sdt_owner.err_mesg_seq;
+CREATE OR REPLACE VIEW sdt_user.glb_par_seq AS SELECT * FROM sdt_owner.glb_par_seq;
+CREATE OR REPLACE VIEW sdt_user.global_parameters AS SELECT * FROM sdt_owner.global_parameters;
+CREATE OR REPLACE VIEW sdt_user.individual_requests AS SELECT * FROM sdt_owner.individual_requests;
+CREATE OR REPLACE VIEW sdt_user.ind_req_seq AS SELECT * FROM sdt_owner.ind_req_seq;
+CREATE OR REPLACE VIEW sdt_user.sdt_ref_seq AS SELECT * FROM sdt_owner.sdt_ref_seq;
+CREATE OR REPLACE VIEW sdt_user.service_requests AS SELECT * FROM sdt_owner.service_requests;
+CREATE OR REPLACE VIEW sdt_user.service_routings AS SELECT * FROM sdt_owner.service_routings;
+CREATE OR REPLACE VIEW sdt_user.service_types AS SELECT * FROM sdt_owner.service_types;
+CREATE OR REPLACE VIEW sdt_user.ser_rou_seq AS SELECT * FROM sdt_owner.ser_rou_seq;
+CREATE OR REPLACE VIEW sdt_user.ser_typ_seq AS SELECT * FROM sdt_owner.ser_typ_seq;
+CREATE OR REPLACE VIEW sdt_user.srv_req_seq AS SELECT * FROM sdt_owner.srv_req_seq;
+CREATE OR REPLACE VIEW sdt_user.target_applications AS SELECT * FROM sdt_owner.target_applications;
+CREATE OR REPLACE VIEW sdt_user.tar_app_seq AS SELECT * FROM sdt_owner.tar_app_seq;
+
+---------------------------------------------------------------
+-- Create Views (instead of Oracle Synonyms) for SDT_BATCH_USER
+---------------------------------------------------------------
+CREATE OR REPLACE VIEW sdt_batch_user.bulk_submissions AS SELECT * FROM sdt_owner.bulk_submissions;
+CREATE OR REPLACE VIEW sdt_batch_user.error_logs AS SELECT * FROM sdt_owner.error_logs;
+CREATE OR REPLACE VIEW sdt_batch_user.global_parameters AS SELECT * FROM sdt_owner.global_parameters;
+CREATE OR REPLACE VIEW sdt_batch_user.individual_requests AS SELECT * FROM sdt_owner.individual_requests;
+--CREATE OR REPLACE VIEW sdt_batch_user.purge AS SELECT * FROM sdt_owner.purge;
+CREATE OR REPLACE VIEW sdt_batch_user.service_requests AS SELECT * FROM sdt_owner.service_requests;
+
+
+------------------------------------------------
+-- Create Grants for SDT_OWNER
+------------------------------------------------
+
+ALTER ROLE sdt_owner SET search_path TO sdt_owner;
+ALTER ROLE sdt_user SET search_path TO sdt_user, sdt_owner;
+ALTER ROLE sdt_batch_user SET search_path TO sdt_batch_user, sdt_owner;
```

Comment:
Do we still need the views with these search_path settings?  For example, can you execute SELECT * FROM sdt_user.bulk_customers as sdt_user without the sdt_user.bulk_customers view defined?  If the views aren't needed then let's remove them.

Reply:
Proven so removing views.